features/step_definitions/snippet_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^I am authenticated$/) do
+  user = User.new email: "landry.soules@gmail.com", password: "1Password", username: "landry"
+  signup user
+  authenticate user
+  page.assert_text "Signed in successfully."
+end
+
+When(/^I fill in the snippet creation form$/) do
+  #TODO: check why cucumber doesn't recognize routing helpers
+  #visit new_snipet
+  visit "/snipets/new"
+  fill_in "snipet_title", :with => "my first snippet"
+  fill_in "snipet_content", :with => "ls"
+  click_button "submit_snipet"
```

Comment:
You're not limited to ids only, you can also use classes or text which is on button (value) for submitting it.